cli/data.go

```diff
@@ -903,6 +903,31 @@ func (c *viamClient) dataRemoveFromDataset(datasetID, orgID, locationID string,
 	return nil
 }
 
+// DataConfigureDatabaseUserConfirmation is the Before action for 'data database configure'.
+// it asks for the user to confirm that they are aware that they are changing the authentication
+// credentials of their database
+func DataConfigureDatabaseUserConfirmation(c *cli.Context) error {
+	yellow := "\033[1;33m%s\033[0m"
+	printf(c.App.Writer, yellow, "WARNING!!!")
+	printf(c.App.Writer, yellow, "If you or someone else in your organization have already created this user, the following steps update the password for that user instead. Dashboards or other integrations relying on this password will then need to be updated.")
```

Comment:
If we can check if a database usere has already been configured, then I'd only print this in that case:
```suggestion
	printf(c.App.Writer, yellow, "You or someone else in your organization have already created a user. The following steps update the password for that user. Once you have updated the password, you will need to update all dashboards or other integrations relying on this password.")
```